Content.Shared/Climbing/Systems/ClimbSystem.cs

```diff
@@ -176,6 +176,9 @@ private void AddClimbableVerb(EntityUid uid, ClimbableComponent component, GetVe
         if (!TryComp(args.User, out ClimbingComponent? climbingComponent) || climbingComponent.IsClimbing || !climbingComponent.CanClimb)
             return;
 
+        if (!component.Vaultable) // Frontier
```

Comment:
There's not anything that has the vaultable value set to false that you're intended to be able to use the "vault" verb on, right? I mean, given the variable name...

Without this it'll give a "vault" verb that silently does nothing.